src/MatchedLearnerApi.Types/TrainingDto.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace MatchedLearnerApi.Types
+{
+    public class TrainingDto
+    {
+        public string Reference { get; set; }
+        public int ProgrammeType { get; set; }
+        public int StandardCode { get; set; }
+        public int FrameworkCode { get; set; }
+        public int PathwayCode { get; set; }
+        public string FundingLineType { get; set; }
+        public DateTime StartDate { get; set; }
+        //public string Status { get; set; } = "TBD";
```

Comment:
use it or lose it - if I remember rightly we decided not to include this, so can be removed. If not then we need to figure out where this comes from and map it through the system

Reply:
removed